crates/store/re_protos/proto/rerun/v1alpha1/manifest_registry.proto

```diff
@@ -135,17 +136,21 @@ message RegisterWithDatasetRequest {
 }
 
 message RegisterWithDatasetResponse {
-  rerun.common.v1alpha1.DataframePart data = 1;
+  rerun.common.v1alpha1.TaskId id = 1;
 }
 
-message RegisterWithDatasetAsyncRequest {
-  // today this just wraps the sync request. This at least avoids
-  // repetition and the two messages going out of sync.
-  RegisterWithDatasetRequest request = 1;
+// TODO(andrea): This is a copy of RegisterDataRequest.
+// Eventually we _may_ get rid of the sync version; until then,
+// we should make sure that the two objects are in sync.
+
```

Comment:
So that TODO comment actually gets associated with this object in the generated code:
```suggestion
```